.github/workflows/ci.yml

```
@@ -44,6 +44,32 @@ jobs:
       - name: before_cache_script
         run: rm -rf $CARGO_HOME/registry/index
 
+  macos-silicon:
+    runs-on: macos-13-aarch64
```

```suggestion
    runs-on: macos-14
```

The new runner is `macos-14`, this is my fault as I have linked the wrong blog [in my previous comment](https://github.com/nix-rust/nix/pull/2311#issuecomment-1935203764), I am sorry, [this post](https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/) should be the right one.

---

done